monai/bundle/scripts.py

```diff
@@ -621,3 +624,80 @@ def ckpt_export(
         more_extra_files=extra_files,
     )
     logger.info(f"exported to TorchScript file: {filepath_}.")
+
+
+def init_bundle(
+    bundle_dir: PathLike,
+    ckpt_file: Optional[PathLike] = None,
+    network: Optional[torch.nn.Module] = None,
+    metadata_str=DEFAULT_METADATA,
```

Comment:
If the default value is `dict`, maybe we should not use `metadata_str` as arg name?
Same as `inference_str`.
I think we may use:
```py
metadata: Union[Dict, str] = DEFAULT_METADATA
```
What do you think?

Thanks.